src/Parser.js

```diff
@@ -670,23 +653,21 @@ Parser.prototype.parseSpecialGroup = function(pos, innerMode, optional) {
  *
  * @return {?ParseFuncOrArgument}
  */
-Parser.prototype.parseGroup = function(pos) {
-    var start = this.lexer.lex(pos, this.mode);
+Parser.prototype.parseGroup = function() {
+    var start = this.nextToken;
     // Try to parse an open brace
     if (start.text === "{") {
```

Comment:
Could just test against `this.nextToken.text` directly instead of introducing `start`